src/sdk/namespace_fs.js

```diff
@@ -2697,4 +2725,23 @@ class NamespaceFS {
 
 }
 
+    // This is helper function for list object version
+    // In order to sort the entries by name we would like to change the name of files with suffix of '_null'
+    // to have the structure of _mtime-...-ino-... as version id.
+    // This function returns a set that contains all file names that were changed (after change)
+    async function _rename_null_version(old_versions, fs_context, version_path) {
+        const renamed_null_versions = new Set();
+        for (const old_version of old_versions) {
+            if (_is_version_null_version(old_version.name)) {
+                const stat = await nb_native().fs.stat(fs_context, path.join(version_path, old_version.name));
```

Comment:
error handling and removal from the array when the error is ENOENT,
need to rethink about other errors